crates/shim-kvm/src/exec.rs

```diff
@@ -12,32 +12,21 @@ use crate::usermode::usermode;
 use core::convert::TryFrom;
 use core::sync::atomic::{AtomicBool, Ordering};
 
+use crate::{
```

Comment:
nit; inconsistent import styling (there's already a `crate` import group at the top)

Reply:
oops.. bad IDE!